k8s-daemonset/k8s/hello-rc.yml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: hello
+spec:
+  replicas: 3
+  selector:
+    app: hello
+  template:
+    metadata:
+      labels:
+        app: hello
+    spec:
+      dnsPolicy: ClusterFirst
+      containers:
+      - name: service
+        image: <helloworld docker image>
+        imagePullPolicy: Always
+        env:
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: POD_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.podIP
+        - name: NS
+          value: helloworld
+        command:
+        - "/bin/bash"
+        - "-c"
+        - "HTTP_PROXY=`./hostIP.sh`:4140 python hello.py"
```

Comment:
Ooooh, nice use of HTTP_PROXY here! Would be great to do the same thing with the internal version of the app.